src/generators/hlopt.ml

```diff
@@ -494,6 +494,7 @@ let code_graph (f:fundecl) =
 				| CNo ->
 					loop (i + 1)
 				| CRet ->
+					assert(b.btrap == []);
```

Comment:
Note that `==` is physical equality. OCaml might optimize this so that all empty lists are of the same instance, but I'm not sure about that. Should be better to use `=` here.

Reply:
You're right, I was reading "=" and typing "==" :( not a good habit for Ocaml.